### DISCUSSION TITLE
What AI models are you using?

### DISCUSSION THREAD

**Discussion by arielweinberger** [pezzolabs/pezzo#11] [Ideas]
The obvious and most common ones are OpenAI's chat completion, DALL-E and text-davinci-003.

I want to know more about your use cases - what AI models are you currently using in production? This is not limited to OpenAI.

**Comment by mikevalstar** [comment]
Currently mostly just gpt3.4 & 4 from openai as of today

However my company is trying multiple

Others we are trying: the recently announced PaLM2 from google, and some self hosted models. 

Hoping to use Pezzo long term to allow quick switching between LLMs, but for now it's a great place to try new prompts and see their results.
